torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -1555,9 +1555,17 @@ def convert(self,
                 load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        # always produce reference pattern for relu
-        is_relu = node.op == "call_function" and node.target == torch.nn.functional.relu
-        if is_reference or is_relu:
+        # always produce reference pattern for following functions
+        func_list = [
+            torch.nn.functional.relu,
+            torch.nn.ReLU6,
+            torch.nn.functional.hardtanh,
+            torch.mean,
+            'relu',
+            'relu_',
+        ]
+        is_func = node.op == "call_function" and node.target in func_list
```

Comment:
is_call_method = node.op == "call_method" and node.target in method_list
is_call_module = node.op == "call_module" and type(modules[node.target]) in module_type_list